gatsby-ssr.js

```
@@ -5,13 +5,34 @@ const PLAUSIBLE_DOMAIN = 'plausible.io';
 const SCRIPT_URI = '/js/script.js';
 
 exports.onRenderBody = ({ setHeadComponents, setHtmlAttributes }) => {
+  const headComponents = [
+    <script
+      key="plain-live-chat"
+      dangerouslySetInnerHTML={{
```

Biome (2.4.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

(lint/security/noDangerouslySetInnerHtml)